crates/pixi_uv_conversions/src/requirements.rs

```diff
@@ -17,20 +17,27 @@ use uv_pep440::VersionSpecifiers;
 use uv_pep508::VerbatimUrl;
 use uv_pypi_types::RequirementSource;
 
-use super::to_git_reference;
+use crate::into_uv_git_reference;
 
 /// Create a url that uv can use to install a version
 fn create_uv_url(
     url: &Url,
-    rev: Option<&GitRev>,
+    rev: Option<&Reference>,
```

Comment:
Now that I'm Looking at it maybe `GitReference` would be a better name especially as its in the global `pixi_spec`?

Reply:
we have already a GitReference in `pixi_git`crate, so to not create a conflict I decided to name this one `Reference` because it lives under `git.rs` so it is related to git by module placement

Reply:
Why can't we reuse that then? And still I dislike seeing reference without namespacing it's just too generic of a word. I would prefer reusing the name and explicitly typing the module.

Reply:
renamed